config/config.properties

```diff
@@ -91,6 +91,21 @@ pravegaservice.dataLogImplementation=BOOKKEEPER
 # Default value: HDFS
 # pravegaservice.storageImplementation=HDFS
 
+# Storage NO-OP Mode: in No-Op mode, user stream segment writing is no-oped; user stream segment reading is not supported.
```

Comment:
This sentence is not obvious: `user stream segment writing is no-oped`. Maybe something like this one (or similar can be clearer for a user): `in No-Op mode, Pravega does not write the payloads of events to Tier 2 storage. Therefore, reads involving data from Tier 2 storage are not supported`.